Formula/php.rb

```diff
@@ -153,6 +159,12 @@ def install
       args << "--with-curl"
     end
 
+    if devel?
```

Comment:
`build.devel?` rather than `devel?` everywhere outside of `test do` blocks, please.